hex/lib/dependabot/hex/credential_helpers.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+module Dependabot
+  module Hex
+    module CredentialHelpers
+      def self.hex_credentials(credentials)
+        organization_credentials(credentials) + repo_credentials(credentials)
+      end
+
+      def self.organization_credentials(credentials)
+        credentials.
+          select { |cred| cred["type"] == "hex_organization" }.
+          flat_map { |cred| cred.slice("type", "organization", "token").values }
```

Comment:
@sorentwo I found a regression in the handling of `hex_organization` credentials. Certain ecosystems, like Hex, can execute arbitrary code during the update process. Because of this, we (GitHub) run the code in Dependabot Core without providing credentials to the container. We need to provide defaults to `organization_credentials/1` in order to account for this. 👇 _should_ do the trick 🙂 

```suggestion
        defaults = { "organization" => "", "token" => "" }
        keys = %w(type organization token)

        credentials.
          select { |cred| cred["type"] == "hex_organization" }.
          flat_map { |cred| defaults.merge(cred).slice(*keys).values }
```